src/components/UserInformationPage/UserTap.js

```diff
@@ -85,12 +97,26 @@ const Message = styled.p`
 `;
 
 export default function UserTap({ user }) {
-  const { email, nickname, imageUrl, followerList } = user;
+  const { _id: userId, email, nickname, imageUrl, followerList } = user;
 
   const [failureReason, setFailureReason] = useState();
+  const [editing, setEditing] = useState(false);
```

Comment:
사용자 정보가 수정되고 있는 상태를 나타내는 state이므로 아래처럼 네이밍하면 조금 더 자연스러울 것 같아요!

```suggestion
  const [isEditing, setIsEditing] = useState(false);
```